src/components/CitizenData.tsx

```diff
@@ -19,12 +26,48 @@ export const CitizenData: React.FunctionComponent<CitizenDataProps> = props => {
   const { t } = useTranslation();
   const [modalState, setModalstate] = useState<boolean>(false);
   const [modalContent, setModalcontent] = useState<string>("");
+  const [resultDataWallet, setResultdataWallet] = useState<Wallet | undefined>(
+    undefined
+  );
 
   function popModal(data: object): void {
     setModalcontent(JSON.stringify(data, null, 3));
     setModalstate(!modalState);
   }
 
+  useEffect(() => {
+    tryCatch(
+      () =>
+        BackofficeClient.GetPMWalletV2({
+          Bearer: `Bearer ${getUserToken()}`,
+          "x-citizen-id": getCitizenId()
+        }),
+      toError
+    )
+      .foldTaskEither(
+        apiError =>
+          fromLeft<Error, TypeofApiResponse<GetPMWalletV2T>>(apiError),
+        apiResponse =>
+          fromEither(
+            apiResponse.mapLeft(err => {
+              return new Error(readableReport(err));
+            })
+          )
+      )
+      .mapLeft(_ => {
+        // TODO: Validation Error
+      })
+      .map(_ => {
+        if (_.status === 200) {
```

Comment:
Should we handle the other status code of this API?
- 404 User not found into the PM
- 429 Rate limit exceeded
- 500 Internal server error